package.json

```diff
@@ -78,6 +78,7 @@
     "randomstring": "^1.1.0",
     "rc": "~0.5.0",
     "request": "^2.51.0",
+    "request-promise": "^1.0.2",
```

Comment:
Basically you just do:

```js
var Promise = require('bluebird')
var request = require('request')
var get = Promise.promisify(request.get)

get(url).spread((resp, body) => ...).catch(err => ...)
```

FYI: in bluebird 3 (which I started using in a different project; here we're still on v2) you have to do `promisify(get, {multiArgs: true})` in order to get both `resp` and `body` into the `then` handler